trackintel/preprocessing/positionfixes.py

```diff
@@ -192,6 +190,9 @@ def generate_triplegs(
         Maximum allowed temporal gap size in minutes. If tracking data is missing for more than
         `gap_threshold` minutes, a new tripleg will be generated.
 
+    print_progress: boolean, default False
+        Show progress bar if set to true, only valid if pfs do not have a column 'staypoint_id' but staypoint are provided.
```

Comment:
I think it would be better to provide descriptions of what the progress bar is showing. Descriptions such as the one provided below: `Assign staypoints to positionfixes`